main.cpp

```diff
@@ -3,30 +3,41 @@
 #include "HybridCar.hpp"
 #include <iostream>
 
+
 int main()
 {
     Car* car = nullptr;
     std::cout << std::endl << "OPEL" << std::endl;
     PetrolCar opel(new PetrolEngine(120, 1800, 6));
     car = &opel;
+    car->Vmax = 200;
```

Comment:
A, tutaj vMax ustawiacie. To powinno być prywatne i w konstruktorze powinno być ustawiane lub od razu bezpośrednio w klasie przy definicji pola.